### PR TITLE
Changing Reward Calculations for Withdrawal Upgrades

### PR DESCRIPTION
**Pull request by TrustHenry**

### CODE REVIEW

**Review by Kang-Simon — APPROVED** [review]
LGTM

File: src/service/schedulers/SupplyAPIScheduler.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ export class SupplyAPIScheduler extends Scheduler {
     private async getRewardBalance(): Promise<any> {
         const client: PoolClient = await pool.connect();
 
-        const SQL = "SELECT SUM(balance) - SUM(balanceactivation) as reward FROM public.validators";
+        const SQL = "SELECT SUM(total_balance) - SUM(effectivebalance) as reward FROM public.validator_balances_p b WHERE b.epoch = (SELECT MAX(epoch) epoch FROM public.epochs e)";
```
Comment (MichaelKim20) [inline src/service/schedulers/SupplyAPIScheduler.ts]:
```suggestion
        const SQL = "SELECT SUM(balance) + SUM(withdrawal) - SUM(balanceactivation) as reward FROM public.validators";
```

**Review by MichaelKim20 — CHANGES_REQUESTED** [review]

File: src/service/schedulers/SupplyAPIScheduler.ts (hunk 1 above)
Comment (TrustHenry) [reply src/service/schedulers/SupplyAPIScheduler.ts]: 반영 하였습니다.

**Review by MichaelKim20 — APPROVED** [review]
LGTM

**Pull request merged**